test-scripts/0.3.0/tests/acceptance.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -ex
+
+curl -i --location --request POST 'http://localhost:8080/report' \
+     --data '@../../../service/src/integration-test/resources/xml/ITK_Report_request.xml' \
```

Comment:
Will have to update this path as the latest is now stored in https://github.com/nhsconnect/integration-adaptor-111/tree/master/service/src/test/resources/xml

Reply:
Updated